### ISSUE TITLE
Setup CI

### ISSUE THREAD

**Issue by Alxandr** [YoloDev/JabbR#1]

**Comment by davepermen** [comment]
really? .. bhahaha

**Comment by Alxandr** [comment]
Yeah. Setting up AppVeyor should be easy. At least if you use msbuild/.slns. Just point it to your sln, and config artifacts => done.

**Comment by ctolkien** [comment]
:heart: AppVeyor

**Comment by davepermen** [comment]
and what's the Point? can it auto-deploy to my clickonce Destination so you can then get the update? i guess not.

**Comment by raymens** [comment]
Maybe if you use Fake (https://gist.github.com/fwaris/a36db5cb21057b95cb50#file-build-fsx)

**Comment by Alxandr** [comment]
Lol. Getting @davepermen to use Fake is probably harder than getting santa to show up for christmas ^^

**Comment by davepermen** [comment]
bhahaha ... we still didn't get auto-clickconce-deployment working, though.. one day.. one day..

**Comment by raymens** [comment]
Deploying is a ftp task?

**Comment by davepermen** [comment]
ish, yes.. it builds properly by now, generates teh files for clickonce (so it builds a clickonce deployment), but the ftp, never got that to work. appveyor weirds me out with those package thingies n stuff.. i just want a filelist where i can select which parts to copy, and then where to.. but no, bundling n stuff n weirdish things.. :dancer:
